FindTown/FindTown/Presentation/MyPageScene/MyPage/MyTownReview/MyTownReviewViewModel.swift

```diff
@@ -13,57 +13,70 @@ import RxRelay
-}
-
 final class MyTownReviewViewModel: BaseViewModel {
     
     struct Input {
         
     }
     
     struct Output {
-        var reviewTableDataSource = BehaviorRelay<[reviewModelTest]>(value: [])
+        var reviewTableDataSource = BehaviorRelay<[ReviewModel]>(value: [])
+        let errorNotice = PublishSubject<Void>()
     }
     
     let input = Input()
     let output = Output()
     let delegate: MyPageViewModelDelegate
     
+    // MARK: - UseCase
+    
+    let authUseCase: AuthUseCase
+    let memberUseCase: MemberUseCase
+    
+    // MARK: - Task
+    
+    private var reviewTask: Task<Void, Error>?
+    
     init(
-        delegate: MyPageViewModelDelegate
+        delegate: MyPageViewModelDelegate,
+        authUseCase: AuthUseCase,
+        memberUseCase: MemberUseCase
     ) {
         self.delegate = delegate
+        self.authUseCase = authUseCase
+        self.memberUseCase = memberUseCase
         
         super.init()
         self.bind()
     }
     
     func bind() {
-        self.output.reviewTableDataSource.accept(returnReviewTestData())
+        self.fetchReview()
+    }
+    
+    func showErrorNoticeAlert() {
+        self.output.errorNotice.onNext(())
     }
 }
 
+// MARK: - NetWork
+
 extension MyTownReviewViewModel {
-    func returnReviewTestData() -> [reviewModelTest] {
-        let demoReview1 = reviewModelTest(village: "서울시 관악구 신림동",
-                                          period: "2년 6개월 거주",
-                                          introduce: "집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 ")
-        let demoReview2 = reviewModelTest(village: "서울시 관악구 신림동",
-                                          period: "2년 6개월 거주",
-                                          introduce: "집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 ")
-        let demoReview3 = reviewModelTest(village: "서울시 관악구 신림동",
-                                          period: "2년 6개월 거주",
-                                          introduce: "집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 ")
-        let demoReview4 = reviewModelTest(village: "서울시 관악구 신림동",
-                                          period: "2년 6개월 거주",
-                                          introduce: "집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 집 앞에 산책 ")
-        
-        let reviews = [demoReview1, demoReview2, demoReview3, demoReview4]
-        return reviews
+    func fetchReview() {
+        self.reviewTask = Task {
+            do {
+                let memberInfo = try await self.memberUseCase.getMemberInfomation(bearerToken: authUseCase.getAccessToken())
```

Comment:
```suggestion
                let  accessToken = authUseCase.getAccessToken()
                let memberInfo = try await self.memberUseCase.getMemberInfomation(bearerToken: accessToken))
```

이렇게 프로퍼티로 분리하는것이 디버깅할때 알아차리기 쉽습니다! 특히나 이렇게 네트워크 로직은 더더 중요해요 
그리고 bearerToken도 수정해볼까요?